packages/terra-table/src/SelectableUtils.js

```diff
@@ -16,21 +29,77 @@ const initialSingleSelectRowIndex = (rows) => {
 };
 
 /**
-* Returns if the selected row index is already selected for SingleSelectableRows.
-*/
+ * Returns the indexes of the selected rows for MultiSelectableRows.
+ * To be used in the constructor, to set initial state.
+ */
+const initialMultiSelectRowIndexes = (rows, maxSelectionCount) => {
+  if (!rows || !rows.length) {
```

Comment:
Since rows are actually React children, we don't need this if check. We can clean this up to be 

`const childArray = React.Children.toArray(row);`

If there are no children, react handles it for us.

Reply:
I think it cleans up 3 physical lines but perhaps forces the reader to mentally read all of the lines? With the existing code, we break out of the function immediately on edge case, and the reader doesn't have to read the rest of the function. Whereas with the new suggestion, React.children.toArray returns an empty function but we can't return just yet. We go to the for loop and then have to know toArray gave us an empty array, and then we skip to return an emptySelectedIndexes.

Code style preference, but explicitly bringing out the edge case seems helpful here.

Reply:
This will likely also trigger in the case that there is only 1 row. React children only become arrays if there is more than one child element. If there is only 1 then it is passed in as type element (so `!rows.length` would be true). I would recommend removing this conditional and just using `React.Children.forEach` for your loop below, or using `toArray` as @emilyrohrbough  suggested. These helper functions will handle all of these edge cases for us.

Reply:
Good point. Learned a bit there! Have adjusted to reflect @emilyrohrbough 's original suggestion =)